Sources/TuistSupport/Vendored/HTTPStatusCodes/HTTPStatusCode+Extensions.swift

```diff
@@ -1,12 +1,3 @@
-// https://github.com/rhodgkins/SwiftHTTPStatusCodes
-//
-//  HTTPStatusCodes+Extensions.swift
-//  HTTPStatusCodes
-//
-//  Created by Richard Hodgkins on 07/06/2016.
-//  Copyright © 2016 Rich H. All rights reserved.
-//
-
```

Comment:
Perhaps we can exclude `/Vendored` or add another `.swiftformat` to exclude the header removal. I believe we should try to retain the copyright notice for cases like this where we have taken partial code / classes from other open source projects.

Reply:
Makes sense!

Reply:
✅